analytical/templatetags/woopra.py

```diff
@@ -23,14 +23,15 @@
      <script type="text/javascript">
       var woo_settings = %(settings)s;
       var woo_visitor = %(visitor)s;
-      (function(){
-        var wsc=document.createElement('script');
-        wsc.type='text/javascript';
-        wsc.src=document.location.protocol+'//static.woopra.com/js/woopra.js';
-        wsc.async=true;
-        var ssc = document.getElementsByTagName('script')[0];
-        ssc.parentNode.insertBefore(wsc, ssc);
-      })();
+      !function(){var a,b,c,d=window,e=document,f=arguments,g="script",h=["config","track","trackForm","trackClick","identify","visit","push","call"],i=function(){var a,b=this,c=function(a){b[a]=function(){return b._e.push([a].concat(Array.prototype.slice.call(arguments,0))),b}};for(b._e=[],a=0;a<h.length;a++)c(h[a])};for(d.__woo=d.__woo||{},a=0;a<f.length;a++)d.__woo[f[a]]=d[f[a]]=d[f[a]]||new i;b=e.createElement(g),b.async=1,b.src="//static.woopra.com/js/w.js",c=e.getElementsByTagName(g)[0],c.parentNode.insertBefore(b,c)}("woopra");
+
+      // configure tracker
+      woopra.config(woo_settings);
+
+      woopra.identify(woo_visitor);
+
+      // track pageview
```

Comment:
It's appreciated that you match the original code from the Woopra documentation, but I feel we can spare the comments and the empty lines (that need to be served to the receiving client). The code is sufficiently self-explanatory without that. Would you mind removing them?